src/models.rs

```diff
@@ -70,7 +70,7 @@ impl GuardrailsHttpRequest {
             return Err(ValidationError::Required("inputs".into()));
         }
         // Validate masks
-        let input_range = 0..self.inputs.len();
+        let input_range = 0..=self.inputs.len();
```

Comment:
Can we add a comment on top of this line explaining why `=` is required for future reader